### DISCUSSION TITLE
live video sprite

### DISCUSSION THREAD

**Discussion by bunsenbeaker** [pixijs/pixijs#7588] [Q&A]
Hi I'm trying to add a **live** streaming video sprite to my pixi app stage.
I tried following the examples in docs - create a Texture from live stream vid URL (in my case 'http://my.server.ip:port/'), create a Sprite from it and add it to stage.
However - I'm getting a frozen image only.
I have tried using PIXI.VideoResource+ base texture etc - but I wasn't able to achieve live video display. When trying to call load() on VideoResource, providing a callback to its Promise's then  - it's never invoked.
I'd appreciate any assistance :)
MK

**Comment by miltoncandelero** [comment]
You _probably_ need to tie the `play`-ish event with a user interaction 🤔 
https://html.spec.whatwg.org/multipage/interaction.html#user-activation-processing-model
one of:

- change
- click
- contextmenu
- dblclick
- mouseup
- pointerup
- reset
- submit
- touchend

**Comment by GreenRefuge** [comment]
I was having a similar issue and realized it was because I had used `PIXI.Ticker.shared.autoStart = false` elsewhere (different part of the app that manages its own Ticker). Removing this fixed the "freeze" for me -- passing along in case it helps :sunglasses: 

I used a local MP4 file when debugging to confirm it wasn't an issue with the "live" video part specifically -- have you tried using a local file with the same result?

Cheers,
